### ISSUE TITLE
Add tab stops to events for 508 compliance

### ISSUE THREAD

**Issue by trutherford2388** [fullcalendar/fullcalendar#5288]
### Bug Reports
When using the fullcalendar it will not have tab stops on the calendar events. With the laws regarding 508 compliance, the events should have a tab stop in order to be 508 compliant.

### Feature Requests
Recommended change is that the links that are used as the events within the calendar have a href attribute added to them to allow for tabbing to each event and then to also include and aria-label attribute that includes the Date and what the event describes.

I.E. ```<a class="fc-day-grid-event fc-h-event fc-event fc-start fc-end" href="#" aria-label="Monday March second 2020, Doctor's Appointment">Doctor's Appointment</a>```

This is not specific to any of the three,  but I am using Vue when I noticed this.

How I am using it in vue is ```<FullCalendar :events="events"
									v-show="this.selectedSchool.SCHOOL_ID !== ''"
									defaultView="dayGridMonth"
									:weekends="calendarWeekends"
									:plugins="calendarPlugins"
									ref="calendar"
									@datesRender="loadEvents" />```

**Comment by acerix** [comment]
Related to https://github.com/fullcalendar/fullcalendar/issues/2536

You could add tab stops using eventRender, eg:

https://codepen.io/acerix/pen/qBdXdzm?editors=0010

**Comment by trutherford2388** [comment]
@acerix, that worked thank you for the help.

**Comment by DCSMaster** [comment]
> 
> 
> Related to #2536
> 
> You could add tab stops using eventRender, eg:
> 
> https://codepen.io/acerix/pen/qBdXdzm?editors=0010

In your render function, if use use 

`info.el.tabIndex = info.event._instance.instanceId;`

It will attach a chronological order to your tab index and control the tab through better

**Comment by BigRaj** [comment]
> 
> 
> Related to #2536
> 
> You could add tab stops using eventRender, eg:
> 
> https://codepen.io/acerix/pen/qBdXdzm?editors=0010

In your render function, if use use

`info.el.tabIndex = info.event._instance.instanceId;`

It will attach a chronological order to your tab index and control the tab through better
